acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
@Magisus I think the code in question from facter 2.x is here https://github.com/puppetlabs/facter/blob/2.x/lib/facter/operatingsystem/base.rb#L15-L54. we essentially would take the `kernelrelease` fact value and split on dots to get major/minor, and just return the `kernelrelease` value for `full`